internal/cmd/environment.go

```diff
@@ -20,12 +29,103 @@ func environmentCmd() *cobra.Command {
 	}
 
 	cmd.AddCommand(environmentDumpCmd())
+	cmd.AddCommand(storeCmd())
 
 	setDefaultFlags(&cmd)
 
 	return &cmd
 }
 
+func storeCmd() *cobra.Command {
+	cmd := cobra.Command{
+		Use:   "store",
+		Short: "Owl store",
+		Long:  "Owl Store",
+	}
+
+	cmd.AddCommand(storeSnapshotCmd())
+
+	return &cmd
+}
+
+func storeSnapshotCmd() *cobra.Command {
+	var (
+		addr      string
+		tlsDir    string
+		sessionID string
+	)
+
+	cmd := cobra.Command{
+		Use:   "snapshot",
```

Comment:
@pastuxso there's a way to set a `Hidden: true` or something like it. Let's do that please as long as it experimental.